src/Compiler.cpp

```diff
@@ -201,108 +287,139 @@ TfLiteNode g_nodes[)"
   for (size_t i = 0; i < nodes_.size(); i++) {
     auto &node = nodes_[i].node;
     auto &regInfo = registrations_[nodes_[i].regIndex];
-    wr.writeBuiltin(regInfo.code, node.builtin_data,
-                    prefix_ + "opdata" + std::to_string(i));
+    if (regInfo.code == tflite::BuiltinOperator_CUSTOM) {
+      wr << "uint8_t " << prefix_ + "opdata" + std::to_string(i) << "[" << node.custom_initial_data_size << "] = { ";
+      for (uint32_t i = 0; i < node.custom_initial_data_size; ++i)
+        wr << int(((uint8_t const*)node.custom_initial_data)[i]) << ", ";
+      wr << " }; /* custom_initial_data */\n";
+    } else {
+      wr.writeBuiltin(regInfo.code, node.builtin_data,
+                      prefix_ + "opdata" + std::to_string(i));
+    }
     wr.writeIntArray(*node.inputs, prefix_ + "inputs" + std::to_string(i));
     wr.writeIntArray(*node.outputs, prefix_ + "outputs" + std::to_string(i));
   }
+  wr << R"(const TensorInfo_t tensors[] = {
```

Comment:
The naming here could be better. Suggestion:

g_tensors -> tflTensors
tensors -> tensorData (or constTensorData if not too verbose)

same for the nodes

Reply:
👍

Reply:
see bug #26